.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,19 @@
+name: golangci-lint
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  golangci-pr:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.38
+          only-new-issues: true
```

Comment:
+ `args: --timeout 5m0s` ?

Reply:
I forgot your patch with the updated config hasn't been merged yet, I added the timeout arg